h2o-r/tests/runitUtils/utilsR.R

```diff
@@ -409,6 +410,18 @@ function(..., envir=parent.frame()) {
     list(tests=tests, envir=envir)
 }
 
+setSeedPriorR36<-
+function(seed) {
+    if (as.numeric(R.Version()$major) >= 4 || 
+        (as.numeric(R.Version()$major) == 3 && as.numeric(substr(R.Version()$minor, 1, 1)) >= 6)
```

Comment:
strictly speaking, to recover the minor version (imagine 3.11.5), you should use sth like
```
as.numeric(unlist(strsplit(R.Version()$minor, '.', fixed=TRUE))[[1]])
```

Reply:
fixed